lib/assert.js

```diff
@@ -256,16 +255,6 @@ function getErrMessage(message, fn) {
   const line = call.getLineNumber() - 1;
   let column = call.getColumnNumber() - 1;
 
-  // Line number one reports the wrong column due to being wrapped in a
-  // function. Remove that offset to get the actual call.
-  if (line === 0) {
-    if (columnOffset === 0) {
-      const { wrapper } = require('internal/modules/cjs/loader');
-      columnOffset = wrapper[0].length;
-    }
-    column -= columnOffset;
-  }
```

Comment:
This part would still be important in case `wrap` or `wrapper` get monkey patched since the line would then not start at an offset of zero. It won't be a big issue though as the assertion will just fall back to the message `false != true`.